diff --git a/node_modules/figma-api/lib/ast-types.d.ts b/node_modules/figma-api/lib/ast-types.d.ts
index c605c90..cc35d7c 100644
--- a/node_modules/figma-api/lib/ast-types.d.ts
+++ b/node_modules/figma-api/lib/ast-types.d.ts
@@ -65,7 +65,9 @@ export declare enum LineHeightUnit {
  * A mapping of a StyleType to style ID (see Style) of styles present on this node. The style ID can be used to look up more information about the style in the top-level styles field.
 */
 export declare type StylesMap = {
-    [styleType in StyleType]: string;
+    effect?: string;
+    fill?: string;
+    text?: string;
 };
 /** Format and size to export an asset at */
 export declare type ExportSetting = {
@@ -276,6 +278,7 @@ declare type EffectShadow_ = {
     blendMode: BlendMode;
     /** How far the shadow is projected in the x and y directions */
     offset: Vector;
+    spread?: number;
 };
 export declare type EffectShadow = {
     type: EffectType.DROP_SHADOW | EffectType.INNER_SHADOW;
@@ -462,7 +465,7 @@ export interface Style {
     /** The name of the style */
     name: string;
     /** The type of style */
-    style_type: StyleType;
+    styleType: StyleType;
 }
 /** The root node */
 export interface DOCUMENT {
