diff --git a/node_modules/figma-api/lib/ast-types.d.ts b/node_modules/figma-api/lib/ast-types.d.ts
index b7c0cfa..4006171 100644
--- a/node_modules/figma-api/lib/ast-types.d.ts
+++ b/node_modules/figma-api/lib/ast-types.d.ts
@@ -65,7 +65,10 @@ export declare enum LineHeightUnit {
  * A mapping of a StyleType to style ID (see Style) of styles present on this node. The style ID can be used to look up more information about the style in the top-level styles field.
 */
 export declare type StylesMap = {
-    [styleType in StyleType]: string;
+    effect?: string;
+    fill?: string;
+    text?: string;
+    grid?: string;
 };
 /** Format and size to export an asset at */
 export declare type ExportSetting = {
@@ -476,7 +479,7 @@ export interface Style {
     /** The name of the style */
     name: string;
     /** The type of style */
-    style_type: StyleType;
+    styleType: StyleType;
 }
 /** The root node */
 export interface DOCUMENT {
@@ -536,6 +539,7 @@ export interface FRAME {
     isMask: boolean;
     /** default: false Does this mask ignore fill style (like gradients) and effects? */
     isMaskOutline: boolean;
+    styles?: StylesMap;
 }
 /** A logical grouping of nodes */
 export declare type GROUP = FRAME;
